tests/unit-test/test_serializer.c

```diff
@@ -511,9 +511,9 @@ int main(void) {
   RUN_TEST(test_serialize_ta_find_transactions_obj_by_tag);
   RUN_TEST(test_recv_mam_message_request_psk_deserialize);
   RUN_TEST(test_recv_mam_message_request_ntru_deserialize);
+  RUN_TEST(test_send_mam_message_request_deserialize);
```

Comment:
Why did you change this file?

Reply:
I just want to keep the order in consistency